p5js/YOLO/sketch.js

```diff
@@ -45,8 +47,8 @@ function startDetecting() {
 }
 
 function detect() {
-  yolo.detect(function(results){
+  yolo.detect(function(results) {
     objects = results;
-    detect();
+    setTimeout(detect, 100);
```

Comment:
Probably should get rid of this? Does this help @cvalenzuela?

Reply:
yep, no need for this!